lib/scenic/adapters/postgres.rb

```diff
@@ -155,17 +155,36 @@ def create_materialized_view(name, sql_definition, no_data: false)
       # @param no_data [Boolean] Default: false. Set to true to create
       #   materialized view without running the associated query. You will need
       #   to perform a non-concurrent refresh to populate with data.
+      # @param side_by_side [Boolean] Default: false. Set to true to create the
+      #   new version under a different name and atomically swap them, limiting
+      #   downtime at the cost of doubling disk usage
       #
       # @raise [MaterializedViewsNotSupportedError] if the version of Postgres
       #   in use does not support materialized views.
       #
       # @return [void]
-      def update_materialized_view(name, sql_definition, no_data: false)
+      def update_materialized_view(
+        name, sql_definition, no_data: false, side_by_side: false
+      )
```

Comment:
I know hound is not going to like this due to line length, but I don't care....

```suggestion
      def update_materialized_view(name, sql_definition, no_data: false, side_by_side: false)
```

Reply:
I'd need to also add some comments to make rubocop be quiet. I don't mind doing so (and actually set `Metrics/LineLength` to 120 on all my projects anyway).

Reply:
I guess rubocop isn't automatically run (and the config file is remarkably old; at least <1.7 and broken with current rubocop versions) so I won't worry about it